pkg/kube/inject/webhook.go

```diff
@@ -589,7 +589,7 @@ func enablePrometheusMerge(mesh *meshconfig.MeshConfig, anno map[string]string)
 	return true
 }
 
-func extractCanonicalServiceLabels(podLabels map[string]string, workloadName string) (string, string) {
+func ExtractCanonicalServiceLabels(podLabels map[string]string, workloadName string) (string, string) {
```

Comment:
i wonder if we should extract this out to a util package to avoid having to import the inject package here?  not sure if that matters or not.